lib/elixir/lib/protocol.ex

```diff
@@ -280,9 +280,9 @@ defmodule Protocol do
     type_args = :lists.map(fn _ -> quote(do: term) end, :lists.seq(2, arity))
     type_args = [quote(do: t) | type_args]
 
-    varify = fn pos -> Macro.var(String.to_atom("arg" <> Integer.to_string(pos)), __MODULE__) end
+    verify = fn pos -> Macro.var(String.to_atom("arg" <> Integer.to_string(pos)), __MODULE__) end
```

Comment:
This one is “correct”, it is a function that converts positions into variables. We should probably call it convert_variables for clarity though.